### PR TITLE
feat:  Adjust minimum desktop width and adjust navigation bar

### LINKED ISSUE #223: Adjust minimum desktop width and adjust navigation bar
### Describe the solution you'd like
Adjust minimum desktop width to 1280px and remove the contribute button to have more space for menu items.

### PR DESCRIPTION
**Pull request by janmichek**
## Description

fixes #223

## Demo
image

## Checklist:

- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [X] Adjust app to new resolution

### CODE REVIEW

File: src/assets/logo.svg
Comment (janmichek) [inline src/assets/logo.svg:1]: On the conference I did rough, unoptimized ad-hoc edit, now I am reusing the logo from Figma

**Review by lukeromanowicz — CHANGES_REQUESTED** [review]
The blockchain panel needs some adjustments. Everything else looks good - nice job!

image

**Review by michele-franchi — CHANGES_REQUESTED** [review]
I get the following layout with the preview link:
image

Could you please check it?

**Comment by janmichek** [comment]
> I get the following layout with the preview link: image
> 
> Could you please check it?

Apparently, it's a problem with the build. I tried to re-run jobs with no luck ./ Please check it on localhost

**Comment by lukeromanowicz** [comment]
@janmichek I'm afraid the issue Michele mentioned is present also in localhost - production build:
image

It's likely caused by modifying display- or flex- related property of a child component in the parent component. It's not showing up in the dev mode because the order of loading components is different than in production build due to HMR being active in dev mode

**Comment by janmichek** [comment]
> @janmichek I'm afraid the issue Michele mentioned is present also in localhost - production build: image
> 
> It's likely caused by modifying display- or flex- related property of a child component in the parent component. It's not showing up in the dev mode because the order of loading components is different than in production build due to HMR being active in dev mode

Ok, now I see where was the problem. I did the wrong rebase. The Contribute button reappeared after messed up rebase. Now everything is in order .)

File: src/components/StatsPanel.vue [hunk 1]
```diff
@@ -218,7 +218,7 @@ const {
 
   @media (--desktop) {
     gap: var(--space-5) var(--space-3);
-    grid-template-columns:248px 248px 285px 285px;
+    grid-template-columns:1fr 1fr 1fr 1fr;
```
Comment (michele-franchi) [inline src/components/StatsPanel.vue:221]:
```suggestion
    grid-template-columns: 1fr 1fr 1fr 1fr;
```

File: src/components/TheNavigation.vue [hunk 2]
```diff
@@ -43,6 +43,16 @@
...
+      <li class="navigation__item">
+        <coming-soon-tooltip>
+          <app-link
+            class="navigation__link navigation__link--disabled"
+            target="_self"
+            to="#">
+            NFTs
+          </app-link>
+        </coming-soon-tooltip>
+      </li>
```
Comment (michele-franchi) [inline src/components/TheNavigation.vue:55]:
```suggestion
```

NFTs should not be in the navigation bar, after the recent change.

**Review by michele-franchi — CHANGES_REQUESTED** [review]
I left some suggestions, please have a look at them.

File: src/components/TheNavigation.vue (hunk 2 above)
Comment (janmichek) [reply src/components/TheNavigation.vue]: True, removed

File: src/components/StatsPanel.vue (hunk 1 above)
Comment (janmichek) [reply src/components/StatsPanel.vue]: fixed

**Review by michele-franchi — COMMENTED** [review]
I checked again and I found one more issue:
image
More keyblocks should be loaded at the beginning to fit the whole horizontal space. Could you increase it?

**Comment by janmichek** [comment]
> I checked again and I found one more issue: image More keyblocks should be loaded at the beginning to fit the whole horizontal space. Could you increase it?

Good catch, fixed

**Review by michele-franchi — APPROVED** [review]
It looks perfect now 👏

**Review by lukeromanowicz — APPROVED** [review]
Looks good to me as well! Well done :+1:

**Review by CharisSiarampalis — APPROVED** [review]

**Pull request merged**